fix: use file-size-loader for glb / gltf files (#26)
moxystudio/next-common-files: README.md

This plugin is meant to handle **3d** files, and tests the file types `.obj`, `.mtl`, `.fnt`, `.gltf` and `.glb`.

> It's not possible to know the size of certain 3D files since the server can't comprress these files right away. To do this it's necessary to use file-size-loader from https://github.com/Odrin/file-size-loader for these types of files.

Reviewer: ```suggestion
It's not possible to know the size of most of these files when you download them, which is useful to show some kind of progress bar. The server delivers them compressed, using gzip or brotli, meaning the response does not contain the `Content-Length` header. To circumvent this problem, we use [`file-size-loader`](https://github.com/Odrin/file-size-loader) instead of the traditional `file-loader`.
```

> An example of the returned object ```example.glb``` would be: 

Reviewer: ```suggestion
An example of the returned object for `example.glb` would be: 
```

> ```javascript

Reviewer: ```suggestion
```js
```

import example from './example.glb'
```

> 

Reviewer: ```suggestion
```

> ```javascript

Reviewer: ```suggestion
```
